account_financial_report_operating_unit/report/open_items.py

```diff
@@ -13,10 +13,10 @@ def _get_report_values(self, docids, data):
 
     @api.model
     def _get_move_lines_domain_not_reconciled(
-        self, company_id, account_ids, partner_ids, target_move, date_from
+        self, company_id, account_ids, partner_ids, only_posted_moves, date_from
```

Comment:
Is this change needed?

Reply:
@yankinmax I want to synchronize the parameter names of the original function.